packages/bot-web-ui/src/components/dashboard/dashboard-component/cards.tsx

```diff
@@ -34,6 +36,22 @@ const Cards = observer(({ is_mobile, has_dashboard_strategies }: TCardProps) =>
     } = dashboard;
     const { handleFileChange, loadFileFromLocal } = load_modal;
     const { loadDataStrategy } = quick_strategy;
+    //this is to check on click of which icon the user has come to bot builder
+    const sentToRudderStackTabChange = (type: string) => {
+        RudderStack.track('ce_bot_builder_form', {
+            shortcut_name: type,
+            form_source: 'bot_dashboard_form-shortcut',
+            action: 'choose_shortcut',
+        });
+    };
+    //this is to check which icon is clicked on dashboard
+    const sentToRudderStack = (type: string) => {
+        RudderStack.track('ce_bot_builder_form', {
+            shortcut_name: type,
+            action: 'choose_shortcut',
+            form_source: 'ce_bot_dashboard_form',
+        });
+    };
```

Comment:
please refactor it proper as previous examples

Reply:
same here